spark/v3.4/spark/src/main/java/org/apache/iceberg/spark/data/vectorized/VectorizedSparkParquetReaders.java

```diff
@@ -84,30 +84,6 @@ public static ColumnarBatchReader buildReader(
                 ColumnarBatchReader::new));
   }
 
-  /**
-   * @deprecated will be removed in 1.3.0, use {@link #buildReader(Schema, MessageType, Map,
-   *     DeleteFilter)} instead.
-   */
-  @Deprecated
-  public static ColumnarBatchReader buildReader(
```

Comment:
There are a few more deprecated methods in this class but they are used in tests to validate set/unset Arrow validity vectors. It is probably a good idea to keep them so that we continue to test that path.